nautobot_ssot_ipfabric/jobs.py

```diff
@@ -51,8 +51,6 @@ def get_formatted_snapshots(client):
     snapshot_refs = []
     if client:
         for snapshot_ref, snapshot in client.snapshots.items():
-            if snapshot.state != "loaded":
```

Comment:
why?

Reply:
In the v6 IPF, only loaded snapshots are returned. Also, the `state` attribute has been removed, so causes an error.